doc-content/enterprise-only/optimizer/download-maven-con.adoc

```diff
@@ -0,0 +1,63 @@
+[id="con-download-maven_{context}"]
+
+= Downloading and configuring the Quarkus Maven repository
+
+If you do not want to use the online Maven repository, you can download and configure the Quarkus Maven repository to create a Quarkus application with Maven. The Quarkus Maven repository contains many of the requirements that Java developers typically use to build their applications.  This procedure describes how to edit the `settings.xml` file to configure the Quarkus Maven repository.
+
+NOTE: When you configure the repository by modifying the Maven `settings.xml` file, the changes apply to all of your Maven projects.
+
+.Procedure
+. Download the {QUARKUS} Maven repository ZIP file from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=redhat.quarkus[Software Downloads] page of the Red Hat Customer Portal (login required).
+. Expand the downloaded archive.
+. Change directory to the `~/.m2/` directory and open the Maven `settings.xml` file in a text editor or integrated development environment (IDE).
+
+. Add the following lines to the `<profiles>` element of the `settings.xml` file, where `_QUARKUS_MAVEN_REPOSITORY_` is the path of the Quarkus Maven repository that you downloaded.  The format of `_QUARKUS_MAVEN_REPOSITORY_` must be `\file://$PATH`, for example `\file:///home/userX/rh-quarkus-{ProductLongVersion}.GA-maven-repository/maven-repository`.
```

Comment:
The `{ProductLongVersion}` variable is undefined.

Reply:
Fixed, thanks